<launch>
  <arg name="camera" default="front_cam" />
  <node name="uvc_camera"  pkg="uvc_camera" type="uvc_camera_node"  output="screen" ns="sedna/camera/front">
    <param name="width" type="int" value="320" />
    <param name="height" type="int" value="240" />
    <param name="fps" type="int" value="30" />
    <param name="frame" type="string" value="wide_stereo" />

    <param name="auto_focus" type="bool" value="False" />
    <param name="focus_absolute" type="int" value="0" />
    <!-- other supported params: auto_exposure, exposure_absolute, brightness, power_line_frequency -->
    <param name="device" type="string" value="/dev/lifecam1" />
    <param name="topic" type="string" value="image_raw" />
<!--    <param name="camera_info_url" type="string" value="file://$(find vision)/launch/front_cam.yaml" /--> 
  </node>
</launch>
